openfisca_tracker/tests/test_piwik.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
Put there, the tests will be packaged in the distribution (unless you [explicitely exclude them](https://github.com/openfisca/openfisca-web-api/blob/master/setup.py#L65)), which is not great. 

Maybe we can just put them in a `tests` folder at the root directory ?